readme.md

```diff
@@ -4,6 +4,9 @@
 ## <a name="A"> </a>A
 ## <a name="B"> </a>B
 ## <a name="C"> </a>C
+
+* [Cyn SMS API SDK](https://github.com/cynojine/cynsms-api) - Cyn SMS API is build for Cyn SMS - Bulk SMS Application For Marketing. **By [@kazashim](https://twitter.com/cynojinetech)**
```

Comment:
Should the description here be "SMS API" ? Should it not be something like composer package. The description appears to be a little bit misleading.